### ISSUE TITLE
Type definition is incompatible with TypeScript 4.9

### ISSUE THREAD

**Issue by TechQuery** [calebdwilliams/element-internals-polyfill#91]
## Expect

Build successfully with TypeScript 4.9:

```json
{
    "compilerOptions": {
        "strict": true,
        "target": "ES5",
        "module": "ES2022",
        "moduleResolution": "Node",
        "jsx": "react-jsx",
        "jsxFactory": "",
        "jsxFragmentFactory": "",
        "lib": ["ES2022", "DOM"]
    },
    "include": ["source/**/*"],
}
```

## Actual

```text
node_modules/.pnpm/element-internals-polyfill@1.1.14/node_modules/element-internals-polyfill/dist/types.d.ts:56:18 - error 
TS2430: Interface 'ICustomElement' incorrectly extends interface 'HTMLElement'.
  The types returned by 'attachInternals()' are incompatible between these types.
    Type 'ElementInternals' is missing the following properties from type 'ElementInternals': ariaColIndexText, ariaInvalid, ariaRowIndexText

56 export interface ICustomElement extends HTMLElement {
                    ~~~~~~~~~~~~~~

 ELIFECYCLE  Command failed with exit code 3.
```

If I rollback to `typescript@4.8.4`, this error disappears.

## Environment

| Software         | Version(s)          |
| ---------------- | ------------------- |
| Node             | 16.18.0             |
| PNPM             | 7.16.0              |
| Operating System | Windows 10 Pro 21H2 |

**Comment by calebdwilliams** [comment]
This should be fixed in the latest version (1.1.18).

**Comment by TechQuery** [comment]
> This should be fixed in the latest version (1.1.18).

@calebdwilliams I think this bug hasn't been fixed:

```text
node_modules/.pnpm/element-internals-polyfill@1.3.11/node_modules/element-internals-polyfill/dist/types.d.ts:60:18 - error TS2430: Interface 'ICustomElement' incorrectly extends interface 'HTMLElement'.
  The types returned by 'attachInternals()' are incompatible between these types.
    Type 'ElementInternals' is missing the following properties from type 'ElementInternals': ariaBrailleLabel, ariaBrailleRoleDescription

60 export interface ICustomElement extends HTMLElement {
                    ~~~~~~~~~~~~~~
```
